spec/seismograph/sensor_spec.rb

```diff
@@ -24,6 +24,21 @@
     end
   end
 
+  context 'when enabled is false' do
+    before do
+      Seismograph.config.enabled = false
+    end
+
+    after do
+      Seismograph.config.enabled = true
```

Comment:
:neckbeard: you could use an `around` here.

```ruby
around do |example|
  Seismograph.config.enabled = false
  example.run
ensure
  Seismograph.config.enabled = true
end
```

The benefit is that if your test raises an exception, then you know it always gets reenabled.